blog/_posts/2017-12-04-multilingual-chatrooms.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: blog-detail
+post-type: blog
+by: Jon Pretty
+title: "Introducing Non-English Chatrooms for Scala Users"
+---
+
+I attend a lot of conferences around the world, and usually all the talks and
+most of the communication among attendees happens in English. It has become (if
+you'll excuse the irony) the *lingua franca* of the Scala community, and the
+wider software industry, even though speakers of English as a first language
+are a minority.
+
+There are, however, many users and potential users of Scala who either do not
+speak English, or do not have the confidence to communicate in public forums in
+English, and the expectation that English be a prerequisite to becoming an
+effective Scala programmer disproportionately handicaps many minority groups
+from countries outside the USA and the rest of the English-speaking world.
+
+So I was surprised to discover that for most non-English languages, there were
+no online communities where people could ask questions in their native tongue.
+And so, it took very little effort to create several channels on
+[Gitter](https://gitter.im/) dedicated to talking about Scala in a variety of
+non-English languages:
+
+- [scala/de](https://gitter.im/scala/de) — Deutsch (German)
+- [scala/es](https://gitter.im/scala/es) — Español (Spanish)
+- [scala/fr](https://gitter.im/scala/fr) — Français (French)
+- [scala/it](https://gitter.im/scala/it) — Italiano (Italian)
```

Comment:
Hey Jon, thank for you this initiative. I think it's very valuable! I just want to report that there's already an Italian-speaking Scala community on Slack, counting a few hundreds people. Do you think there's value in keeping a gitter channel too, or can we maybe just link to the Slack channel?

Reply:
No, I suspect not... the same is true of the Polish channel.